src/overlays/actors/ovl_Obj_Kendo_Kanban/z_obj_kendo_kanban.c

```diff
@@ -1,28 +1,48 @@
 /*
  * File: z_obj_kendo_kanban.c
  * Overlay: ovl_Obj_Kendo_Kanban
- * Description: Swordsman's School - Cuttable Board
+ * Description: Swordsman's School - Cuttable Board at back of school
  */
 
 #include "z_obj_kendo_kanban.h"
 #include "objects/object_dora/object_dora.h"
 
 #define FLAGS 0x00000000
 
+// Board Fragment Flags: Identify which of the 4 quadrants fo the board are present.
+#define PART_FULL 0
+#define PART_TOP_RIGHT (1 << 0)
+#define PART_TOP_LEFT (1 << 1)
+#define PART_BOTTOM_RIGHT (1 << 2)
+#define PART_BOTTOM_LEFT (1 << 3)
+#define RIGHT_HALF (PART_TOP_RIGHT | PART_BOTTOM_RIGHT)
+#define LEFT_HALF (PART_TOP_LEFT | PART_BOTTOM_LEFT)
+#define TOP_HALF (PART_TOP_RIGHT | PART_TOP_LEFT)
+#define BOTTOM_HALF (PART_BOTTOM_RIGHT | PART_BOTTOM_LEFT)
+
+// Number of bounces the board takes before settling.
+#define MAX_BOUNCE_COUNT (7)
+
 #define THIS ((ObjKendoKanban*)thisx)
 
+typedef enum {
+    /* -1 */ DIR_DOWN = -1,
+    /*  0 */ DIR_UNDETERMINED,
+    /*  1 */ DIR_UP,
+} BoardDirection;
```

Comment:
```suggestion
typedef enum {
    /* -1 */ OBJKENDOKANBAN_DIR_DOWN = -1,
    /*  0 */ OBJKENDOKANBAN_DIR_UNDETERMINED,
    /*  1 */ OBJKENDOKANBAN_DIR_UP,
} ObjKendoKanbanDirection;
```
Prefixing enums/structs with the actor's name is preferred to help avoid name collisions. Could you also do the same for the `BoardFragmentFlags` up above?